README.md

```diff
@@ -186,36 +190,43 @@ Switch application:
 - [ ] deactivate monit for old application
 - [ ] shut down the old application
 
-Switch cron jobs:
+Switch cron jobs when there is a gap in schedule:
 - [ ] clear cron jobs on old server
-- [ ] install cron jobs on new server
-- [ ] update post-receive hook to install cronjobs
+  - [ ] immediately install cron jobs on new server
+- [ ] update post-receive hook on new server to install cronjobs
 
 Expand Letsencrypt cert on new server:
-- [ ] Configure new nginx to listen do production domain
-- [ ] Forward http traffic from old to new server
-- [ ] /usr/local/share/letsencrypt/env/bin/letsencrypt certonly -w /etc/letsencrypt/webrootauth/ -d prod2.ceresfairfood.org.au -d members.ceresfairfood.org.au --expand
+- [ ] Configure new nginx to listen to production domain
+- [ ] Proxy pass http (not https) traffic from old to new server (so that letstencrypt can generate cert)
+- [ ] `/usr/local/share/letsencrypt/env/bin/letsencrypt certonly -w /etc/letsencrypt/webrootauth/ -d prod2.ceresfairfood.org.au -d shop.ceresfairfood.org.au --expand`
 
 Switch databases:
-- [ ] check databases are in sync
-- [ ] make new mysql writable
-- [ ] change database.yml
+- [ ] check databases are in sync (eg check your user updated_at after visiting a page)
+- [ ] make new mysql db writable
+- [ ] change new database.yml to point to local db server
 - [ ] stop delayed job
+- [ ] confirm no other scheduled tasks currently running or about to run
+- [ ] Set old master to read-only
 - [ ] reload new application
 - [ ] make new mysql master `STOP SLAVE; RESET MASTER;`
```

Comment:
Depending on the research above, we may need to deactivate any persisted master-slave config for restarts.